pages/common/shards.md

```diff
@@ -0,0 +1,23 @@
+# shards
+
+> Dependency management tool for the Crystal language.
+
+- Create a skeleton shard.yml file:
+
+`shards init`
+
+- Install dependencies from shard.yml file:
```

Comment:
Perhaps ...`from a shard.yml`... would read better?

Reply:
Made the change at 6ac0440 !